plugins/pulp_rpm/plugins/migrations/0020_nested_drpm_directory.py

```diff
@@ -112,6 +112,11 @@ def migrate(*args, **kwargs):
     """
     Fix the drpm directories and broken repomd.xml files.
     """
+    # skip this migration if we have no drpm units
+    drpm_collection = get_collection('units_drpm')
+    if drpm_collection.count() == 0:
+        return
```

Comment:
A log statement here at INFO would be valuable, just to have a record of whether this path happened.